### DISCUSSION TITLE
Still not get Copilot pack

### DISCUSSION THREAD

**Discussion by kazeokamali** [community/community#146304] [GitHub Education]
### Hi everyone,

Hello,everyone,
it has been 5days since I receive this 'Approved'message, but I still cant use the student pack.
Anyway to check the status?
![Snipaste_2024-12-05_08-52-25](https://github.com/user-attachments/assets/c3b18585-2269-48c4-b3a0-e64ec67b3025)

**Comment by davevad93** [comment]
Hi @kazeokamali ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @kazeokamali was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by kazeokamali** [comment]
i‘ve got that，thanks…💓💕

---- Replied Message ----
| From | Davide ***@***.***> |
| Date | 12/05/2024 21:13 |
| To | community/community ***@***.***> |
| Cc | Fang ***@***.***>,
Mention ***@***.***> |
| Subject | Re: [community/community] Still not get Copilot pack (Discussion #146304) |

Hi @kazeokamali ,

When the benefits become available, the status label will turn purple and change to:

Congrats! Your request for @kazeokamali was approved and your benefits are now available.

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions:
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the Student Developer Pack Application FAQs :

How long after I submit my application will it be approved?
How long after I’ve been approved will I receive my academic benefits?

**Reply by davevad93** [reply]
You're welcome @kazeokamali 👍
